### PR TITLE
Fixed invalid link in model_pruning readme

### PR DESCRIPTION
**Pull request by cenh**
Current link to deep cnn is invalid and leads to the start page for tutorials.

### CODE REVIEW

**Comment by cenh** [comment]
I signed it!

File: tensorflow/contrib/model_pruning/README.md [hunk 1]
```diff
@@ -133,7 +133,8 @@ For now, it is assumed that the underlying hardware platform will provide mechan
 
 ## Example: Pruning and training deep CNNs on the cifar10 dataset <a name="example"></a>
 
-Please see https://www.tensorflow.org/tutorials/deep_cnn for details on neural
+Please see [Advanced Convolutional Neural Networks
+](https://www.tensorflow.org/tutorials/images/deep_cnn) for details on neural
```
Comment (penpornk) [inline tensorflow/contrib/model_pruning/README.md]: Could you please move the closing bracket to the previous line?
```
Please see [Advanced Convolutional Neural Networks]
(https://www.tensorflow.org/tutorials/images/deep_cnn) for details on neural
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for your PR and I'm so sorry for our delay!

File: tensorflow/contrib/model_pruning/README.md (hunk 1 above)
Comment (cenh) [reply tensorflow/contrib/model_pruning/README.md]: Moved things around, see dd0a1b2

**Review by penpornk — APPROVED** [review]
Thank you for contributing!

**Pull request merged**